static/src/javascripts/projects/common/views/experiments/measure-understanding.html

```diff
@@ -0,0 +1,12 @@
+<div class="understanding">
+    <div class="understanding__text">Do you feel better informed as a result of reading this piece?</div>
+    <button class="understanding__choice js-button" data-value="yes">
+        <%= thumbsUp %>
+        <span>Yes</span>
+    </button>
+    <button class="understanding__choice js-button" data-value="no">
+        <%= thumbsDown %>
+        <span>No</span>
+    </button>
+    <div class="understanding__feedback" hidden>Thank you!</div>
```

Comment:
You could do this based on `[aria-hidden]`, which gives you the accessibility benefit, plus it works in IE < 11 :)

Plus: `<div>` -> `<p>` aswell ..